### DISCUSSION TITLE
Allow sending HTML in send email flow operation

### DISCUSSION THREAD

**Discussion by raflymln** [directus/directus#14492] [Archive]
### Describe the Bug

As seen on:
https://github.com/directus/directus/blob/857eae54defc166bef6198d482bdaba2a0b876cf/api/src/operations/mail/index.ts#L17-L21

md() function is actually sanitize html so every time I tried to send and html, the attributes on the html tag would always stripped out

---

So for this bug, I'd like to propose to add one selection input under the subject input to choose whether the content must be send as html or md, or you could provide either markdown or wysiwyg editor for the body section.

I think this "bug" should be done as soon as possible since it can be a game changer for the send email function

### To Reproduce

I tried putting these into the body under the send mail operation
```html
<section style="color: #8d8c87; display: block; font-family: Arial, sans-serif; font-size: 12px; line-height: 120%; text-align:center;">
  
  <center>
   <img src="https://somedomain.com/assets/assetId.png" alt="logo" height="50">
  </center>
  
  # Hi! Welcome to Our Newsletter!
  
  Hi {{ $trigger.payload.email }}! Thank you for your enthusiasm for joining our newsletter list :D
  
</section>
```

Now, if you see on the preview tab, it's all looking fine, until you send them and you check them on the email

And the result received on the email was 

```html
<section>
<h1>Hi! Welcome to Our Newsletter!</h1>
</section>
```

### Errors Shown

No error shown

### What version of Directus are you using?

v9.13.0

### What version of Node.js are you using?

17.0.4

### What database are you using?

10.7.4-MariaDB-1:10.7.4+maria~bionic - mariadb.org binary distribution

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/14363
